doc/api/policy.md

```diff
@@ -109,5 +109,76 @@ In order to generate integrity strings, a script such as
 `printf "sha384-$(cat checked.js | openssl dgst -sha384 -binary | base64)"`
 can be used.
 
+Integrity can be specified as the boolean value `true` in order to accept any
+body for the resource which can be useful for local development. It is not
+recommended in production since it would allow unexpected alteration of
+resources to be considered valid.
+
+### Dependency Redirection
+
+An application may need to ship patched versions of software or to prevent
+software from allowing all modules access to all other modules. In order to
+do so redirection can be used.
+
+```json
+{
+  "builtins": [],
+  "resources": {
+    "./app/checked.js": {
+      "dependencies": {
+        "fs": true,
+        "os": "./app/node_modules/alt-os"
+      }
+    }
+  }
+}
+```
+
+The dependencies are keyed by the requested string specifier and have values
+of either `true` or a string pointing to a module that will be resolved.
+
+The specifier string does not perform any searching and must match exactly
+what is provided to the `require()`. Therefore, multiple specifiers may be
```

Comment:
Not currently while both are experimental. It can apply to them as we move forward unblocking things. As it stands both have some refactoring going on so I'm not going to be trying to keep it up to date except to ensure that no designs prevent adding policies in the future.